hg-fast-export.sh

```diff
@@ -81,7 +81,6 @@ do
       # pass --force to git-fast-import and hg-fast-export.py
       GFI_OPTS="$GFI_OPTS --force"
       IGNORECASEWARN="";
-      break
```

Comment:
But what happens if the user adds a flag which is passed on to `hg-fast-export.py` before the `--force`, then force will be passed on to fast-export, but no to fast import? That can't be right.